js/modules/k6/http/http.go

```diff
@@ -60,6 +60,14 @@ func (r *RootModule) NewModuleInstance(vu modules.VU) modules.Instance {
 		}
 	}
 
+	validateArgCount := func(methodName string, args ...sobek.Value) {
+		numberOfArgs := len(args)
+		if numberOfArgs > 2 {
+			vu.State().Logger.Warningf("http.%s method has more than two arguments, but %d were provided",
```

Comment:
Nit; I'd prefer if we can use the "actionable" version of the message suggested by @olegbespalov 👉🏻 [here](https://github.com/grafana/k6/pull/3974#discussion_r1796491116).